lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -199,77 +185,14 @@ public SystemAccessControl create(Map<String, String> config)
         {
             requireNonNull(config, "config is null");
 
-            Bootstrap bootstrap = new Bootstrap(
-                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class));
+            FileBasedSystemAccessControlModule module = new FileBasedSystemAccessControlModule(config);
```

Comment:
It is good to have refactoring changes in separate commits, so it is easier to track what actually has changed.